algorithm/obstats/snow/ims_snow_template.yaml.j2

```diff
@@ -0,0 +1,47 @@
+- obs space:
+    name: ims_snow
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: {{ snow_obsdatain_path }}/snow/diag_{{ obspace }}_{{ stat_current_cycle_YMDH }}.nc
+    simulated variables: ['totalSnowDepth']
+    observed variables: ['totalSnowDepth']
+  statistics to compute: ['mean', 'count', 'RMS']
+  output file: "{{ obspace }}_{{ stat_current_cycle_YMDH }}_output_snow.nc"
+  domains to process:
+  - domain:
+      name: "nowhere"
```

Comment:
Just a question on the domain name "nowhere".  Is that specified elsewhere or could it be named something else?  'Global' or 'none' would be my preference if it can be renamed, if it can't I'll cope.

Reply:
I agree, this should be 'global' or 'all' since it should include all observations around the globe

Reply:
@EdwardSafford-NOAA @CoryMartin-NOAA Good catch and I agree with you both. The domains were from the original prototype yaml.j2 file for testing and I had not paid much attention to the actual domain names themselves.